README.md

```diff
@@ -6,6 +6,8 @@
 
 Tutorial for using the Vision OCV integration of Klepsydra.
 
+The [chapter1](./tutorial/chapter1.md) covers the explanation of the vision_ocv module of kpsr-robotics (using vision_ocv and mem_dlw).
```

Comment:
> (using vision_ocv and mem_dlw) 

Replace this with : This chapter explains the core API of the vision_ocv module and its usage.

Reply:
Done